Source/ACE.Server/WorldObjects/WorldObject_Use.cs

```diff
@@ -92,12 +93,27 @@ public virtual void OnActivate(WorldObject activator)
                 return;
             }
 
+            if (player != null)
+                player.EnchantmentManager.StartCooldown(this);
+
             // if ActivationTarget is another object,
             // should this be checking the ActivationResponse of the target object?
 
-            // default use action
-            if (ActivationResponse.HasFlag(ActivationResponse.Use))
+            // Chests with ActivationResponse.CastSpell ( and Active 65535 ) should
+            // still process ActOnUse as normal but need to stop further processing
+            // after completing, as the CastSpell portion is handled internally
+            // by the Chest() class only during the Chest().Open event
+            if (WeenieType == WeenieType.Chest && Active == Active.SecondaryAction)
```

Comment:
this is not needed, and is definitely not related to Active=0xFFFF

the patch posted in the dev channel already had this working without any specialized logic required